apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Snippets/BigQueryClientSnippets.cs

```diff
@@ -1428,5 +1428,50 @@ public void CreatePartitionedTable()
         // TODO:
         // Dataset update/patch
         // Table update/patch
+
+        [Fact]
+        public void DryRunValidQuery()
+        {
+            string projectId = _fixture.ProjectId;
+            string datasetId = _fixture.GameDatasetId;
+            string tableId = _fixture.HistoryTableId;
+
+            // Sample: DryRunValidQuery
+            BigQueryClient client = BigQueryClient.Create(projectId);
+            BigQueryTable table = client.GetTable(projectId, datasetId, tableId);
+            CreateQueryJobOptions options = new CreateQueryJobOptions { DryRun = true };
+            BigQueryJob job = client.CreateQueryJob($"SELECT player, game_started FROM {table}", options);
+            // There are no rows in the result, but we do have the schema as if we'd run the query.
+            TableSchema schema = job.Resource.Statistics.Query.Schema;
+            foreach (var field in schema.Fields)
+            {
+                Console.WriteLine($"{field.Name}: {field.Type}");
+            }
+            // End sample
```

Comment:
Assert that no result is returned, to make sure that the DryRun has really been applied?